### PR TITLE
MIDI cleanup (2/2)

### PR DESCRIPTION
**Pull request by dreamer**
Moving files around to bring some order in here :)

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -12,12 +12,21 @@ endif
...
                hardware/serialport/libserial.a \
+               ints/libints.a \
                libs/decoders/libdecoders.a \
                libs/gui_tk/libgui_tk.a \
                libs/nuked/libnuked.a \
-               libs/ppscale/libppscale.a
+               libs/ppscale/libppscale.a \
+               midi/libmidi.a \
+               misc/libmisc.a \
+               shell/libshell.a
```
Comment (kcgen) [inline src/Makefile.am:30]: :abc: I like the ordering.

File: vs/dosbox.vcxproj.filters [hunk 2]
```diff
@@ -1,814 +1,823 @@
...
     <Filter Include="src\shell">
-       <UniqueIdentifier>{d7f601fb-7919-4adc-8a23-cc5c8a14ff94}</UniqueIdentifier>
+      <UniqueIdentifier>{d7f601fb-7919-4adc-8a23-cc5c8a14ff94}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="src\midi">
+      <UniqueIdentifier>{5920d562-013f-417e-8095-198288c97b52}</UniqueIdentifier>
     </Filter>
   </ItemGroup>
-</Project>
+</Project>
```
Comment (kcgen) [inline vs/dosbox.vcxproj.filters:823]: (maybe missing a CR?)

Comment (dreamer) [reply vs/dosbox.vcxproj.filters:823]: I updated Visual Studio, perhaps it now decided to no longer use newline for its XML files…

**Comment by dreamer** [comment]
Thanks for the review :) This opens the field for initial testing of FluidSynth integration. More cleanups might be in order, but they will be bundled with FluidSynth work from now on.

**Pull request merged**